ros2_generate_interface_docs/ros2_generate_interface_docs/templates/msg.html.em

```diff
@@ -0,0 +1,42 @@
+@{
+import time
+}@
+<html>
+  <head>
+    <title>@(interface_name) Documentation</title>
+    <link type="text/css" rel="stylesheet" href="../../msg-styles.css" />
+  </head>
+  <body>
+  <div id="container">
+    <h1 class="msg-title"><a href="../index-msg.html">@(interface_package)</a>/@(interface_name) @(type)</h1>
+    <div class="msg">
+      <h2>File: <span class="filename">@(interface_package)/@(interface_name).@(ext)</filename></h2>
+      <h2>Raw Message Definition</h2>
+      <div class="raw-msg">
+      @[for line in raw_text.splitlines()]@
+        @{parts = line.split('#')}@
```

Comment:
If you are ever only interested in two parts you can avoid unnecessary splitting and then rejoining by passing the max number to split: `line.split('#', 1)`